app/containers/App/index.js

```diff
@@ -10,6 +10,7 @@ import React from 'react';
 import Helmet from 'react-helmet';
 import styled from 'styled-components';
 import { FormattedMessage } from 'react-intl';
+import PropType from 'prop-types';
```

Comment:
`import PropTypes from 'prop-types';` please use this.

Reply:
Ahhhh okay Alright sure